autodepgraph/tests/test_graph.py

```diff
@@ -53,6 +53,13 @@ def test_maintain_node_assume_unkown_is_good(self):
         self.assertEqual(self.test_graph.nodes()['C']['state'], 'good')
         self.assertEqual(self.test_graph.nodes()['B']['state'], 'unknown')
 
+    def test_calibration_state(self):
```

Comment:
What do you expect to get back? I don't know what the method is supposed to do and the test does not make this any clearer.

Reply:
Trying to make coverage happy ;-)